Qubit Server/dev/src/org/labrad/qubits/QubitContext.java

```diff
@@ -509,6 +500,38 @@ public void mem_sync_delay() {
       memDirty = true;
   }
   //
+  // Jump Table
+  //
+
+  @Setting(id = 370,
+           name = "Clear Jump Table",
+           doc = "Clear the jump table commands in this context.")
+  public void clear_jump_table() {
+    getExperiment().clearJumpTable();
+  }
+
+  @Setting(id = 371,
+           name = "Add Jump Table Entry",
+           doc = "Add a jump table entry to a single board.")
+  public void add_jump_table_entry(@Accepts({"s", "(ss)"}) Data channel,
+                                   @Accepts("s") Data command_name,
```

Comment:
This one can just be `String command_name`. Makes for less type-checking later (though you may want to check the value against some enum).